components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Protocol/src/Stack/Encoders/JsonEncoderEx.cs

```diff
@@ -1,14 +1,7 @@
-/* Copyright (c) 1996-2016, OPC Foundation. All rights reserved.
-   The source code in this file is covered under a dual-license scenario:
-     - RCL: for OPC Foundation members in good-standing
-     - GPL V2: everybody else
-   RCL license terms accompanied with this source code. See http://opcfoundation.org/License/RCL/1.00/
-   GNU General Public License as published by the Free Software Foundation;
-   version 2 of the License are accompanied with this source code. See http://opcfoundation.org/License/GPLv2
-   This source code is distributed in the hope that it will be useful,
-   but WITHOUT ANY WARRANTY; without even the implied warranty of
-   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
-*/
```

Comment:
I'm not an expert in licenses, but imho this is derivative works and the license header needs to stay in place.  --> Better check with legal before repurposing license terms.

Reply:
I'm not either, I trusted Marc's judgement here https://github.com/Azure/Industrial-IoT/issues/535 and wanted to help. whatever ...

Reply:
yeah, well, if Marc double checked it with legal its ok

---
In reply to: [440824348](https://github.com/Azure/Industrial-IoT/pull/555#discussion_r440824348) [](ancestors = 440824348)